mistral-search-engine.ipynb

```diff
@@ -0,0 +1,721 @@
+       "<pre>\n",
+       "aiosignal           1.3.1\n",
+       "annotated_types     0.7.0\n",
+       "anyio               NA\n",
+       "appnope             0.1.4\n",
+       "asttokens           NA\n",
+       "async_timeout       4.0.3\n",
+       "attr                23.2.0\n",
+       "certifi             2024.06.02\n",
+       "charset_normalizer  3.3.2\n",
+       "comm                0.2.2\n",
+       "cython_runtime      NA\n",
+       "dateutil            2.9.0\n",
+       "debugpy             1.6.7\n",
+       "decorator           5.1.1\n",
+       "exceptiongroup      1.2.0\n",
+       "executing           2.0.1\n",
+       "frozenlist          1.4.1\n",
+       "h11                 0.14.0\n",
+       "httpcore            1.0.5\n",
+       "httpx               0.25.2\n",
+       "idna                3.7\n",
+       "ipykernel           6.29.4\n",
+       "jedi                0.19.1\n",
+       "multidict           6.0.5\n",
+       "orjson              3.10.4\n",
+       "packaging           24.0\n",
+       "parso               0.8.4\n",
+       "pickleshare         0.7.5\n",
+       "pkg_resources       NA\n",
+       "platformdirs        4.2.2\n",
+       "prompt_toolkit      3.0.47\n",
+       "psutil              5.9.0\n",
+       "pure_eval           0.2.2\n",
+       "pydantic            2.7.3\n",
+       "pydantic_core       2.18.4\n",
+       "pydev_ipython       NA\n",
+       "pydevconsole        NA\n",
+       "pydevd              2.9.5\n",
+       "pydevd_file_utils   NA\n",
+       "pydevd_plugins      NA\n",
+       "pydevd_tracing      NA\n",
+       "pygments            2.18.0\n",
+       "pytz                2024.1\n",
+       "six                 1.16.0\n",
+       "sniffio             1.3.1\n",
+       "soupsieve           2.5\n",
+       "stack_data          0.6.2\n",
+       "swig_runtime_data4  NA\n",
+       "tornado             6.3.3\n",
+       "traitlets           5.14.3\n",
+       "typing_extensions   NA\n",
+       "urllib3             2.2.1\n",
+       "vscode              NA\n",
+       "wcwidth             0.2.13\n",
+       "yarl                1.9.4\n",
+       "zmq                 25.1.2\n",
+       "zoneinfo            NA\n",
+       "</pre>\n",
+       "</details> <!-- seems like this ends pre, so might as well be explicit -->\n",
+       "<pre>\n",
+       "-----\n",
+       "IPython             8.25.0\n",
+       "jupyter_client      8.6.2\n",
+       "jupyter_core        5.5.0\n",
+       "-----\n",
+       "Python 3.10.14 (main, May  6 2024, 14:42:37) [Clang 14.0.6 ]\n",
+       "macOS-14.4-arm64-arm-64bit\n",
+       "-----\n",
+       "Session information updated at 2024-06-11 03:48\n",
+       "</pre>\n",
+       "</details>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 81,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import session_info\n",
```

Comment:
!pip install session_info is maybe missing?

Reply:
session-info is only to display the libraries version, it is not necessary required.

Reply:
Its just that it makes it more user friendly to be able to just download the notebook and run it on colab without any major changes, if possible.

Reply:
no problem I will remove it.